internal/connector/pingone_platform/resources/pingone_notification_settings.go

```diff
@@ -0,0 +1,44 @@
+package resources
+
+import (
+	"github.com/pingidentity/pingctl/internal/connector"
+	"github.com/pingidentity/pingctl/internal/logger"
+)
+
+// Verify that the resource satisfies the exportable resource interface
+var (
+	_ connector.ExportableResource = &PingoneNotificationSettingsResource{}
+)
+
+type PingoneNotificationSettingsResource struct {
+	clientInfo *connector.SDKClientInfo
+}
+
+// Utility method for creating a PingoneNotificationSettingsResource
+func NotificationSettings(clientInfo *connector.SDKClientInfo) *PingoneNotificationSettingsResource {
+	return &PingoneNotificationSettingsResource{
+		clientInfo: clientInfo,
+	}
+}
+
+func (r *PingoneNotificationSettingsResource) ExportAll() (*[]connector.ImportBlock, error) {
+	l := logger.Get()
+
+	l.Debug().Msgf("Fetching all %s resources...", r.ResourceType())
+
+	importBlocks := []connector.ImportBlock{}
+
+	l.Debug().Msgf("Generating Import Blocks for all %s resources...", r.ResourceType())
+
+	importBlocks = append(importBlocks, connector.ImportBlock{
+		ResourceType: r.ResourceType(),
+		ResourceName: "notification",
```

Comment:
nit: maybe `notification_settings` for the name

Reply:
now have this edited as well as `branding` to `branding_settings`